wallet/hardware.md

```diff
@@ -82,6 +83,31 @@ This type of set-up does come with additional hurdles that may be more difficult
 
 ***
 
+## Jade
+
+The Blockstream [Jade](https://blockstream.com/jade/) is one of the smallest hardware wallets on the market but offers a lot of features like the support of SeedQR and a built-in battery. Beginners or people who are not interested in the us of stateless devices have the option to use USB, Bluetooth or SeedQR with a stored seed in combination with a virtual secure element (pin server). There is also the possibility to use the Jade as stateless signing device only. 
+
+<p align="center">
+<img src="/assets/img/jade.png" class=responsive width="350" height="350" maxheight="350">
+</p>
+
+### Notable Features
+* Fully air-gapped operation possible
+* Stateless operation possible
+* Fully open source
+
+
+### Notable Limitations
+* No physical secure element but the possibility to save your seed on the device in combination with a virtual secure element *(not mandatory)*
```

Comment:
Would be good to link to docs on the PIN server here so people fully understand the trade-offs

Reply:
done with https://github.com/BitcoinQnA/Bitcoiner.Guide/pull/39/commits/200e6a036ca1638bc37953c58d56e7ffea62c896